### PR TITLE
Added Samson to the teams page

### PR DESCRIPTION
**Pull request by samswag**
Added my Bio to the company's page.

### CODE REVIEW

File: company/team/index.md [hunk 1]
```diff
@@ -398,6 +398,13 @@ To add yourself to this page:
...
 
+## Samson Goddy (he/him)
+- Director of Community
+- Port Harcourt, Nigeria 🇳🇬
+- GitHub: [samswag](https://github.com/samswag)
+- [samson@sourcegraph.com](mailto:samson@sourcegraph.com),[@samson_goddy](https://twitter.com/Samson_Goddy),[LinkedIn](https://linkedin.com/in/samsongoddy)
+- Samson, a software developer who believes in creating something iconic. An open source advocate who has been sustaining projects with his diverse experiences. He maintains Tools, Desktops and likes contributing back to the large OSS ecosystem. Building communities is something he passionate about, and he currently runs Open Source Community Africa, a non-profit org that promotes and educates everything open source. He loves playing console games, mobile photography and loves eating jollof rice.
```
Comment (mrnugget) [inline company/team/index.md:406]:
```suggestion
- Samson, a software developer who believes in creating something iconic. An open source advocate who has been sustaining projects with his diverse experiences. He maintains tools, desktops and likes contributing back to the large OSS ecosystem. Building communities is something he passionate about, and he currently runs Open Source Community Africa, a non-profit org that promotes and educates everything open source. He loves playing console games, mobile photography and eating jollof rice.
```

File: company/team/index.md [hunk 2]
```diff
@@ -435,7 +442,7 @@ To add yourself to this page:
...
-- Eli was raised in Baltimore, MD, has been in New York City since 2007. He has been in SaaS sales with early stage to high growth companies for a majority of his professional career. Prior to joining Sourcegraph, Eli was an Enterprise Account Executive at Segment, and previously at Mixpanel during similar stages of company growth. When not listening to the latest tech and business podcasts, Eli is a student of U.S and World history, enjoys playing chess, following his favorite sports teams, and traveling.
+- Eli was raised in Baltimore, MD, has been in New York City since 2007 (resided in 4 of the 5 boroughs), and recently joined the unofficial 6th borough of NYC, Boca Raton, FL. He has been in SaaS sales with early stage to high growth companies for a majority of his professional career. Prior to joining Sourcegraph, Eli was an Enterprise Account Executive at Segment, and previously at Mixpanel during similar stages of company growth. When not listening to the latest tech and business podcasts, Eli is a student of U.S and World history, enjoys playing chess, following his favorite sports teams, and traveling with his wife.
```
Comment (mrnugget) [inline company/team/index.md:445]: Are these changes supposed to be in here? 😄

**Review by mrnugget — APPROVED** [review]
Approve, but I think you might need to rebase onto `main`, because there's other changes in here that I'm not sure you wanted to make.

File: company/team/index.md (hunk 2 above)
Comment (samswag) [reply company/team/index.md:445]: I am not sure what happened, lol. I was making an edit on Github.

**Comment by mrnugget** [comment]
Closing because this would need to be rebased but it was pushed to a fork, so I can't rebase it right now and Samson is on his phone :D

**Pull request closed without merging**